infra/dockerfile.commit_retrieval

```diff
@@ -4,18 +4,18 @@ COPY infra/git_hyper_blame_patch /root/git_hyper_blame_patch
 
 # Mercurial need Python2 :(
 # git and git-hyper-blame (from depot_tools) are required by the annotate pipeline.
-RUN apt-get update && \
-    apt-get install -y python python-pip git curl && \
+RUN apk update && apk upgrade && \
+    apk add --no-cache gcc g++ libgomp lapack-dev git libexecinfo-dev make libc6-compat && \
```

Comment:
Are you sure you need gcc here? It wasn't needed before

Reply:
Yes. Mercurial setup requires gcc to be installed.
I'll still look if I can reduce some dependency

Reply:
Mmmh, that's strange, how is it working without your changes?

Reply:
Not sure, but python package earlier was may be sufficient to compile headers required for mercurial install.

Reply:
Oh, So earlier slim image used to install gcc as a part of "apt-get update" command

`The following additional packages will be installed:
  binutils build-essential bzip2 cpp cpp-6 dbus dirmngr dpkg-dev fakeroot file
  g++ g++-6 gcc gcc-6`

While this isn't the case with alpine image

Reply:
Can you check if you can run the `commit_retriever` script in a Docker container with this image?

Reply:
I got some errors installing numpy in bug-http_service. 
I'll update you as soon as I resolve them